android/src/main/java/com/reactlibrary/RNNativeLogsModule.java

```diff
@@ -29,37 +30,52 @@ public String getName() {
     private File getFile(String fileIdentifier) {
         File path = this.reactContext.getFilesDir();
         String pathname = path + "/" + fileIdentifier + ".txt";
-        File file = new File(pathname);
-        return file;
+        return new File(pathname);
     }
 
     @ReactMethod
-    public void setUpRedirectLogs(String fileIdentifier, final Promise promise) {
+    public void setUpRedirectLogs(String fileIdentifier, String logLevel, final Promise promise) {
```

Comment:
This a breaking change for iOS. You should keep the old signature as well (the one without the log level). iOS does not have the logLevel signature so will fail. Same for the others methods